apps/wallet/src/ui/app/components/accounts/AccountMultiSelectItem.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useResolveSuiNSName } from '@mysten/core';
+import { CheckFill16, SocialGoogle24 } from '@mysten/icons';
+
+import * as ToggleGroup from '@radix-ui/react-toggle-group';
+import classNames from 'classnames';
+import { AccountItem } from './AccountItem';
+
+type AccountMultiSelectItem = {
+	address: string;
+	name?: string;
+	disabled?: boolean;
+	selected: boolean;
+};
+
+type AccountMultiSelectItemProps = {
+	address: string;
+	name?: string;
+	disabled?: boolean;
+	selected: boolean;
+};
+
+export function AccountMultiSelectItem({
+	address,
+	selected,
+	disabled,
+	...props
```

Comment:
might be a bit more clear to use the `name` directly and override the domainName in `AccountItem` name prop